types/formidable/index.d.ts

```diff
@@ -184,6 +184,13 @@ declare namespace formidable {
          */
         multiples?: boolean | undefined;
 
+        /**
+         * If true, makes direct folder uploads possible.
+         *
+         * @default false
+         */
+        createDirsFromUploads?: boolean;
```

Comment:
```suggestion
        createDirsFromUploads?: boolean | undefined;
```
(to stick with already applied convention) thx!